attractions/package.json

```diff
@@ -43,13 +45,14 @@
     "rollup-plugin-terser": "^7.0.2",
     "sass": "^1.30.0",
     "stylelint": "^13.6.1",
-    "svelte": "^3.24.0",
+    "svelte": "^3.29.1",
     "svelte-preprocess": "^4.5.2",
     "typescript": "^4.0.5"
   },
   "peerDependencies": {
     "postcss": ">=7 <9",
     "sass": "^1.30.0",
+    "svelte": "^3.29.1",
```

Comment:
Should this be mentioned in the changelog?

Reply:
Yes, I was even thinking if this is a major change or not. I suppose it shouldn't be, because keeping your dependencies up-to-date is always a good idea

Reply:
No, this is definitely not a breaking change. It is just that we documented it